polygeist/tools/cgeist/Test/Verification/sycl/raising/host_raising_captures.cpp

```diff
@@ -30,14 +30,17 @@ class KernelName;
 // CHECK:         sycl.host.set_captured %[[lambda]][3] = %[[bar_const]] : !llvm.ptr, i64
 
 // COM: float/double vectors are lowered to native MLIR types; again, just ensure that the value we marked comes from the CGF.
```

Comment:
```suggestion
// COM: vectors are passed as pointers to a struct; check that we matched the corresponding memcpy.
```

Reply:
This is the new diff:
```
$ git diff upstream/sycl sycl/test-e2e/Matrix/element_wise_all_ops_impl.hpp
diff --git a/sycl/test-e2e/Matrix/element_wise_all_ops_impl.hpp b/sycl/test-e2e/Matrix/element_wise_all_ops_impl.hpp
index e394059d2b3b..61d0b4fbc5aa 100644
--- a/sycl/test-e2e/Matrix/element_wise_all_ops_impl.hpp
+++ b/sycl/test-e2e/Matrix/element_wise_all_ops_impl.hpp
@@ -71,8 +71,7 @@ void verify_op_a(const T l, const T r, const float ref, OP op) {
 
            ext::intel::experimental::matrix::joint_matrix_store(
                sg, sub_mat,
-               accessMat.template get_multi_ptr<access::decorated::no>() +
-                   (sg_startx * SUB_ROWS) * NUM_COLS +
+               accessMat.get_pointer() + (sg_startx * SUB_ROWS) * NUM_COLS +
                    sg_starty / SG_SZ * SUB_COLS,
                NUM_COLS);
          }); // parallel for
@@ -110,12 +109,11 @@ void verify_op_c(const T l, const T r, const float ref, OP op) {
              wi_slice[i] = op(wi_slice[i], r);
            }
 
-           joint_matrix_store(
-               sg, sub_mat,
-               accessMat.template get_multi_ptr<access::decorated::no>() +
-                   (sg_startx * SUB_ROWS) * NUM_COLS +
-                   sg_starty / SG_SZ * SUB_COLS,
-               NUM_COLS, layout::row_major);
+           joint_matrix_store(sg, sub_mat,
+                              accessMat.get_pointer() +
+                                  (sg_startx * SUB_ROWS) * NUM_COLS +
+                                  sg_starty / SG_SZ * SUB_COLS,
+                              NUM_COLS, layout::row_major);
          }); // parallel for
```
Basically `accessMat.template get_multi_ptr<access::decorated::no>()` is changed to `accessMat.get_pointer()`.

Reply:
done.